### ISSUE TITLE
pulse: Deconv runtime output diverges at second chunk boundary (kernel > stride)

### ISSUE THREAD

**Issue by JulienBalianSonos** [sonos/tract#2203]
On tract main (commit `b29780d79`, no patches), pulse-mode Deconv (ConvTranspose) with `kernel > stride` produces wrong values starting at the boundary of the second chunk. The graph **types** through pulse correctly: `output_facts` are right, `PulsedFact` propagation is fine. The runtime values diverge from the bulk reference.

## Minimal reproducer

t2n-exported model from:

```python
import torch, torch.nn as nn
from torch_to_nnef import TractNNEF, export_model_to_nnef

class M(nn.Module):
    def __init__(self):
        super().__init__()
        self.up = nn.ConvTranspose1d(8, 8, kernel_size=32, stride=16)
    def forward(self, x):
        return self.up(x)

m = M().eval()
x = torch.randn(1, 8, 4)
target = TractNNEF(version='0.23.0-dev.5', check_io=False, dynamic_axes={'x': {2: 'T'}})
export_model_to_nnef(m, args=(x,), file_path_export='/tmp/just_up.nnef.tgz', inference_target=target,
    input_names=['x'], output_names=['y'])
```

```
tract /tmp/just_up.nnef.tgz --nnef-tract-core --nnef-tract-pulse --pulse 'T=2' --hint 'T=8' compare --stream --allow-random-input
```

```
At turn 0, wrong value for output 0, Mismatch.
First outlier: Close for F32) at [0, 0, 32] 0.02849944 != 0.01945249.
Outliers: 744 / 1024 = 0.72656 > 0.00000.
```

Index `32` is the start of the second pulse (16 frames per pulse `*` stride=16 = 32-sample boundary). The first 32 samples (one pulse) match the reference; everything after diverges, suggesting the Deconv's pulse-state buffer (overlap-add) isn't being initialised / accumulated correctly across calls.

## Reproduces with

- Depthwise (`groups=in_channels`) and dense (`groups=1`).
- Various `(kernel, stride)` combinations where `kernel > stride`.
- Both with and without [#2201](https://github.com/sonos/tract/pull/2201) and [#2202](https://github.com/sonos/tract/pull/2202) applied (this is independent of those type-level fixes).

## Context

Surfaced while implementing pulse-mode export for Kyutai Pocket-TTS' Mimi audio decoder ([sonos/torch-to-nnef PR #76](https://github.com/sonos/torch-to-nnef/pull/76)). After landing the LCM-merge type-correctness fix (#2202), the graph types and pulsifies cleanly, but `compare --stream` reveals this runtime divergence.

## Suspected location

`pulse/src/ops/cnn/deconv.rs` and/or `pulse-opl/src/deconv.rs` (the overlap-add state machinery for Deconv).

**Comment by JulienBalianSonos** [comment]
## Root cause

Bias double-counting in the overlap region.

The pulse-mode `Deconv` calls per-pulse `Deconv` with `adjustments = stride - 1` and `padding = Valid`, producing `(P-1)*S + K + (S-1)` output slots. The bulk `Deconv`'s `DeconvSum` initializes the output buffer with the broadcasted bias, then accumulates frame contributions on top. So per-pulse output already has bias added to *every* slot, including the `K - S` slots in the overlap region that the bulk graph would never emit.

`DeconvDelay`'s overlap-add then sums pulse N's last `K - 1` slots (with bias) into pulse N+1's first `K - 1` slots (also with bias). Result: every overlap slot gets `2 * bias`.

Why existing tests miss it: every `Deconv` test in `harness/core-proptest-pulse/src/deconv.rs` uses `rctensor0(0f32)` (scalar zero) for bias via `DeconvOp::chain` (line 32). With `bias = 0`, the double-add is a no-op.

## Fix (#2204)

Feed a zero bias to the per-pulse `Deconv`, then add the original bias broadcast to `(1, ..., C, ..., 1)` post-`DeconvDelay` so it lands exactly once per output position. PR #2204 also adds two pure-Rust regressions (dense and depthwise, K=32, S=16, non-zero bias).
